Dockerfile

```diff
@@ -2,4 +2,9 @@ FROM python:3.5
 RUN mkdir /var/run/sshd
 RUN chmod 0755 /var/run/sshd
 RUN apt-get update && apt-get install -y libffi-dev ssh
+RUN apt-get install -y unzip && \
+    wget https://releases.hashicorp.com/terraform/0.11.3/terraform_0.11.3_linux_amd64.zip && \
+    unzip terraform_0.11.3_linux_amd64.zip && \
+    mv terraform /usr/local/bin/
 RUN pip3 install tox
+WORKDIR /dcos-launch
```

Comment:
? this is just a "cd dcos-launch" when you "docker run"

Reply:
yeah, but there is no `dcos-launch` directory here unless you mount one at the command line. For example, I mount volumes to docker containers very often and I just always mount them as `/build` regardless of their name. If I try doing that with this docker image, there will be an issue when the workdir is `dcos-launch` but there is no `dcos-launch` dir because I didn't make one. How would I (or any other user), just looking to use this image off the shelf know that we must run the image always with something mounted at `/dcos-launch`?

Reply:
I see. I've always ran the command from the README with `pwd:...` so didn't think about that!